CppCoreGuidelines.md

```diff
@@ -18881,6 +18881,12 @@ Doing so takes away an `#include`r's ability to effectively disambiguate and to
         copy(/*...*/);    // now overloads local ::copy and std::copy, could be ambiguous
     }
 
+##### Note
+
+An exception is `using namespace std::literals;`. This is necessary to use string literals
+in header files and given [the rules](http://eel.is/c++draft/over.literal), users are required 
+to name their own UDLs `operator""_x`, they will not collide with the standard library.
```

Comment:
This reads a little oddly to me. I think it would be better if "users are required to name their own UDLs `operator""_x`" was in parentheses, or surrounded by dashes:

> This is necessary to use string literals in header files and given [the rules](http://eel.is/c++draft/over.literal) &mdash; users are required to name their own UDLs `operator""_x` &mdash; they will not collide with the standard library.